code/datums/diseases/berserker.dm

```diff
@@ -0,0 +1,49 @@
+/datum/disease/berserker
+	name = "Berserker"
+	max_stages = 2
+	spread_text = "Non-Contagious"
+	spread_flags = SPECIAL
+	cure_text = "Anti-Psychotics"
+	cures = list("haloperidol")
+	agent = "Anger."
+	viable_mobtypes = list(/mob/living/carbon/human,/mob/living/carbon/monkey)
+	desc = "Swearing, shouting, attacking nearby crew members uncontrollably."
+	severity = DANGEROUS
+	disease_flags = CURABLE
+	spread_flags = NON_CONTAGIOUS
+
+/datum/disease/berserker/stage_act()
+	..()
+	var/mob/living/M = affected_mob
+	switch(stage)
+		if(1)
+			if(prob(5))
+				M.emote(pick("twitch", "grumble"))
+			if(prob(5))
+				var/speak = pick("Grr...", "Fuck...", "Fucking...", "Fuck this fucking.. fuck..")
+				M.say(speak)
+		if(2)
+			if(prob(5))
+				M.emote(pick("twitch", "scream"))
```

Comment:
use picklists for this shit like brain damage does

get rid of the switch and use an if, if, if fallthrough so you're not replicating code over and over